examples/lighting-app/nrfconnect/README.md

```diff
@@ -432,6 +435,13 @@ page.
 
 ## Flashing and debugging
```

Comment:
Please change `flashing` to `programming` to be consistent with the Nordic terminology. This is a global change for the whole file.